sdk/typespec/typespec_client_core/src/http/policies/retry/mod.rs

```diff
@@ -302,10 +304,7 @@ mod test {
         let retry_after = get_retry_after(
             &headers,
             datetime_now,
-            &[
-                (HeaderName::from_static("x-ms-retry-after"), false),
-                (RETRY_AFTER, true),
-            ],
+            &[HeaderName::from_static("x-ms-retry-after"), RETRY_AFTER],
```

Comment:
Nit (if there are other changes): may as well just define this as a `const` in the `tests` module instead of repeating it, but this is fine too.